### PR TITLE
Login routes, component and mobile version added

### PR DESCRIPTION
**Pull request by kiraigg**

### CODE REVIEW

File: client/src/components/Register/index.jsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ export const Register = () =>
                         <input type="password" id="password-input" placeholder="4+ characters"/>
                     </div>
                     <button className="btn btn-primary sing-up">sign up</button>
+                    <a href="#">Already have an account?</a>
```
Comment (InnaKutsik) [inline client/src/components/Register/index.jsx]: Тег <Link> з роутерів і роут на логін

Comment (kiraigg) [reply client/src/components/Register/index.jsx]: fixed

File: client/src/components/ForgotPassword/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import {Link} from "react-router-dom";
+
+export const ForgotPassword = () =>
+    <div className="login-container ">
+        <div className="container-fluid">
+            <div className="bg-img">
+                <div className="logo-in-sign-up">
+                    <h1>Sport News</h1>
+                </div>
+                <img src={login_bg} alt="login_bg"/>
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/index.jsx]: change alt pls

File: client/src/components/ForgotPassword/index.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                <div className="get-started">
+                    <h3>Don't have an account?</h3>
+                    <Link to="/register" className="btn btn-primary">Get Started</Link>
+                </div>
+                <div className="forgot-password-wrapper">
+                    <h2>Forgot your password?</h2>
+                    <p>Enter your email address below and we'll get you back on track.</p>
+                    <div className="email-wrapper">
+                        <label htmlFor="email-input">Email address</label>
+                        <input type="email" id="email-input" placeholder="Email@gmail.com"/>
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/index.jsx]: we can use only gmail email?

File: client/src/components/ForgotPassword/style.scss [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+
+    .get-started {
+      display: flex;
+      align-items: center;
+      justify-content: space-around;
+      width: 100%;
+      height: 83px;
+      max-width: 375px;
+      padding: 30px 0;
+      margin: 0 20px 0 auto;
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: Interesting margin. How does this `auto` influence on space on the left?

File: client/src/components/ForgotPassword/style.scss [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+
+      @media screen and (max-width: $mobile-width) {
+        display: none;
+      }
+    }
+
+    .forgot-password-wrapper {
+      width: 100%;
+      max-width: 400px;
+      margin: 200px 0 auto;
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: same qn

File: client/src/components/ForgotPassword/style.scss [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+          color: $havelock;
+          text-align: right;
+
+          @media screen and (max-width: $mobile-width) {
+            margin-top: 20px;
+            color: $white;
+          }
+        }
+
+        @media screen and (max-width: $mobile-width) {
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: order of style should be next
1) main styles
2) styles in media queries
3) children, siblings and so on


so, in this case, it is hard to find selector for this media

File: client/src/components/ForgotPassword/style.scss [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+        font-size: 12px;
+        font-style: normal;
+        font-weight: bold;
+        line-height: 14px;
+        letter-spacing: 1px;
+        color: $white;
+        text-transform: uppercase;
+      }
+
+      @media screen and (max-width: $mobile-width) {
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: who is the father of this media?

File: client/src/components/Login/index.jsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+import {Link} from "react-router-dom";
+
+export const Login = () =>
+    <div className="login-container ">
+        <div className="container-fluid">
+            <div className="bg-img">
+                <div className="logo-in-sign-up">
+                    <h1>Sport News</h1>
+                </div>
+                <img src={login_bg} alt="login_bg"/>
```
Comment (anna-kuzii) [inline client/src/components/Login/index.jsx]: alt

File: client/src/components/Login/style.scss [hunk 9]
```diff
@@ -0,0 +1,269 @@
+.login-container {
+  .container-fluid {
```
Comment (anna-kuzii) [inline client/src/components/Login/style.scss]: this styles repeated a lot of times. Maybe the is a reason to move it to general file. how do u think?

File: client/src/components/Login/style.scss [hunk 10]
```diff
@@ -0,0 +1,269 @@
...
+    display: flex;
+    justify-content: flex-start;
+    max-width: 1440px;
+    padding: 0;
+    background: $white;
+  }
+
+  .bg-img {
```
Comment (anna-kuzii) [inline client/src/components/Login/style.scss]: same

File: package.json [hunk 11]
```diff
@@ -27,15 +30,16 @@
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "bootstrap": "^4.3.1",
+    "concurrently": "^4.1.2",
```
Comment (anna-kuzii) [inline package.json]: what does this lib do?

File: client/src/components/ForgotPassword/index.jsx (hunk 3 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/index.jsx]: nope, it's from mockup :)

File: package.json (hunk 11 above)
Comment (kiraigg) [reply package.json]: allows you to start server and client with one command in 1 terminal

File: client/src/components/ForgotPassword/style.scss (hunk 4 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: pushing block with button "Get started" to the right side

File: client/src/components/ForgotPassword/style.scss (hunk 5 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: removed

File: client/src/components/ForgotPassword/style.scss (hunk 6 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: reworked

File: client/src/components/Login/index.jsx (hunk 8 above)
Comment (kiraigg) [reply client/src/components/Login/index.jsx]: renamed

File: client/src/components/Login/style.scss (hunk 9 above)
Comment (kiraigg) [reply client/src/components/Login/style.scss]: done

File: client/src/components/ForgotPassword/index.jsx (hunk 2 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/index.jsx]: done

File: client/src/components/ForgotPassword/style.scss [hunk 12]
```diff
@@ -0,0 +1,84 @@
...
+      font-size: 12px;
+      font-style: normal;
+      font-weight: bold;
+      line-height: 14px;
+      letter-spacing: 1px;
+      color: $white;
+      text-transform: uppercase;
+    }
+
+    @media screen and (max-width: $mobile-width) {
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: move it to the selector

File: client/src/components/ForgotPassword/style.scss [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+      max-width: 80%;
+      margin: 200px 0 auto;
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: refactor it

File: client/src/components/ForgotPassword/style.scss [hunk 14]
```diff
@@ -0,0 +1,84 @@
+.login-container {
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: why login-container if it is forgot password?
if it is a common class, then choose better name

File: client/src/components/styles/_forms.scss [hunk 15]
```diff
@@ -0,0 +1,241 @@
...
+      background: $crimson;
+    }
+  }
+}
+
+.create-account-wrapper,
+.log-in-wrapper {
+  width: 100%;
+  max-width: 400px;
+  margin: 200px 0 auto;
```
Comment (anna-kuzii) [inline client/src/components/styles/_forms.scss]: same

File: client/src/components/styles/_forms.scss [hunk 16]
```diff
@@ -0,0 +1,241 @@
...
+
+  @media screen and (max-width: $mobile-width) {
+    max-width: 80%;
+    margin: 200px 0 auto;
```
Comment (anna-kuzii) [inline client/src/components/styles/_forms.scss]: refactor

File: client/src/components/ForgotPassword/style.scss [hunk 17]
```diff
@@ -0,0 +1,84 @@
...
+  .forgot-password-wrapper {
+    width: 100%;
+    max-width: 400px;
+    margin: 200px 0;
+
+    ::placeholder {
```
Comment (anna-kuzii) [inline client/src/components/ForgotPassword/style.scss]: I believe we have these styles are in general file

Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: fixed

File: client/src/components/styles/_forms.scss (hunk 16 above)
Comment (kiraigg) [reply client/src/components/styles/_forms.scss]: done

File: client/src/components/ForgotPassword/style.scss (hunk 14 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: reworked

File: client/src/components/ForgotPassword/style.scss (hunk 13 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: done

File: client/src/components/ForgotPassword/style.scss (hunk 12 above)
Comment (kiraigg) [reply client/src/components/ForgotPassword/style.scss]: moved

**Pull request merged**